src/assets/css/table.css

```diff
@@ -1,119 +1,183 @@
-
 .table {
     margin-bottom: 1px;
-}
 
-.table *::selection{
-    background-color: transparent;
-    color: $brand-font-color;
-}
+     *::selection {
+        background-color: transparent;
+        color: $brand-font-color;
+    }
 
-.table .input-body-container *::selection {
-    background-color: $brand-color-primary;
-    color: #fff;
-}
+    .input-body-container *::selection {
+        background-color: $brand-color-primary;
+        color: #fff;
+    }
 
-.table tbody+tbody {
-    border-top: 0;
-}
+    &.table-included {
+        background-color: transparent;
+        height: 100%;
+        
+        td {
+            height: 22px;
+            width: 50%;
+        }
+    }
 
-.table thead th {
-    border: 0;
-    background-color: $brand-color-secondary;
-    color: #fff;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    overflow: hidden;
-    vertical-align: middle;
-}
+    &.table-bordered-horizontally td {
+        border-bottom: 1px solid $brand-border-color;
+    }
 
-.table thead .sort-menu .th-caption {
-    text-overflow: ellipsis;
-    overflow: hidden;
-    display: inline-block;
-    width: 100%;
-}
+    &.table-bordered-vertically td:not(:last-child) {
+        border-right: 1px solid $brand-border-color;
+    }
 
-.table .col-xs-12,
-.table .col-sm-12 {
-    position: static;
-}
+    &.table-bordered-vertically .item-caption td {
+        border-right: none;
+    }
 
-.table.table-bordered-horizontally td {
-    border-bottom: 1px solid $brand-border-color;
-}
+    &.table-bordered-vertically td.indented {
+        line-height: 28px;
+        min-width:0;
```

Comment:
Just a small nitpick, there are some spaces missing here. You can leave them like this until we introduce a tool to automatically fix CSS formatting